templates/download/server/thank-you-s390x.html

```diff
@@ -1,52 +1,122 @@
 {% extends "download/_base_download.html" %}
 
 {% block title %}Thanks for downloading Ubuntu Server for IBM Z and LinuxONE{% endblock %}
+
 {% block canonical_url %}https://ubuntu.com/download/server/s390x{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1GptXtMfbgOjOmqb8DZCCTRwogkiP_cM95U9Z0gFSt7g/edit{% endblock meta_copydoc %}
 
```

Comment:
missing description

Reply:
Updated. Strange that it was showing when I did it this way 
<img width="1430" alt="Screenshot 2024-04-17 at 13 25 04" src="https://github.com/canonical/ubuntu.com/assets/14939793/cd9359c9-c895-47b9-9238-301e65ec8a2e">